src/sys/unix/ready.rs

```diff
@@ -126,9 +126,7 @@ impl UnixReady {
 
     #[cfg(not(any(target_os = "dragonfly",
         target_os = "freebsd", target_os = "ios", target_os = "macos")))]
-    #[deprecated(since = "0.6.12", note = "this function is now platform specific")]
-    #[doc(hidden)]
-    pub fn aio() -> UnixReady {
+    fn aio() -> UnixReady {
```

Comment:
Why is this fn still needed?

Reply:
It's used below in the Debug impl. Note, it's not public.